File: SecondYear/anshv-dev_AnshVerma_2226cseai1184_2/Week_2/fibonacci.cpp
vector<int> generateFibonacciNumbers(int n) {
    // Write your code here.
    if(n==1){
        return {0};
    }
    else if(n==2){
        return {0,1};
    }
    vector<int> fib(n,0);
    fib[1]=1;
    for(int i=2;i<n;i++){
        fib[i] = fib[i-1]+fib[i-2];
    }
    return fib;
}

File: SecondYear/aayushraghav93_AayushRaghav_2226CSE(AI)1108_2/Week_2/Fibonacci.java
public class Fibonacci {
    public static int[] generateFibonacciNumbers(int n) {
        // Write your code here.
        if (n <= 0) {
            return new int[0]; 
        }

        int[] fibonacciSequence = new int[n];
        fibonacciSequence[0] = 0; 

        if (n > 1) {
            fibonacciSequence[1] = 1; 
        }

        for (int i = 2; i < n; i++) {
            fibonacciSequence[i] = fibonacciSequence[i - 1] + fibonacciSequence[i - 2];
        }

        return fibonacciSequence;
    }
}